test/test-sets/ref/active_cover.stderr

```diff
@@ -10,23 +10,23 @@ power_t = 0.5
 Enabled reductions: gd, active_cover, scorer-identity, binary, count_label
 Input label = simple
 Output pred = scalar
-average  since         example        example        current        current  current 
-loss     last          counter         weight          label        predict features 
-0.000000 0.000000            1            1.0         1.0000         1.0000       63 
-0.500000 1.000000            2            2.0         1.0000        -1.0000      147 
-0.250000 0.000000            4            4.0        -1.0000        -1.0000      142 
-0.250000 0.250000            8            8.0         1.0000         1.0000       22 
-0.125000 0.000000           16           16.0        -1.0000        -1.0000       50 
-0.156250 0.187500           32           32.0         1.0000         1.0000       85 
-0.140625 0.125000           64           64.0         1.0000        -1.0000      190 
-0.140625 0.140625          128          128.0         1.0000         1.0000      134 
-0.156250 0.171875          256          256.0         1.0000         1.0000       54 
+average  since         example        example        current        current  current
+loss     last          counter         weight          label        predict features
+0.000000 0.000000            1            1.0         1.0000         1.0000       63
+0.500000 1.000000            2            2.0         1.0000        -1.0000      147
+0.250000 0.000000            4            4.0        -1.0000        -1.0000      142
+0.250000 0.250000            8            8.0         1.0000         1.0000       22
+0.125000 0.000000           16           16.0        -1.0000        -1.0000       50
+0.156250 0.187500           32           32.0         1.0000         1.0000       85
+0.140625 0.125000           64           64.0         1.0000        -1.0000      190
+0.140625 0.140625          128          128.0         1.0000         1.0000      134
+0.156250 0.171875          256          256.0         1.0000         1.0000       54
 
 finished run
 number of examples = 500
 weighted example sum = 500.000000
 weighted label sum = -10.000000
 average loss = 0.156000
-best constant = -0.020000
-best constant's loss = 0.999600
+best constant = -0.040005
```

Comment:
interesting that this bug wasn't reported before

Reply:
Yea that is weird, I assume the test wasn't intending on changing the loss_function